radsat-sk/operation/subsystems/RAntenna.h

```diff
@@ -7,18 +7,28 @@
 #ifndef RANTENNA_H_
 #define RANTENNA_H_
 
+#include <stdint.h>
+
 /***************************************************************************************************
                                             DEFINITIONS
 ***************************************************************************************************/
 
-/** I2C Slave Address for Transceiver Receive Port */
-#define ANTENNA_I2C_SLAVE_ADDR (0x31)
+/** I2C Slave Address for Antenna Receive Port */
+#define ANTENNA_I2C_SLAVE_ADDR_PRIMARY (0x31)

```

Comment:
Comment says recieve port, the name says primary, but it's the only one - are there supposed to be two? Or just one? The comments/name could be more clear